include/ginkgo/core/matrix/dense.hpp

```diff
@@ -1148,6 +1148,24 @@ class Dense
         return create_submatrix(rows, columns, this->get_stride());
     }
 
+
+    /**
+     * Create a submatrix from the original matrix.
+     *
+     * @param rows     row span
+     * @param columns  column span
+     * @param global_size size of the submatrix (only used for consistency with
+     *                    distributed::Vector)
```

Comment:
```suggestion
     * @param rows  row span
     * @param columns  column span
     * @param size  size of the submatrix (only used for consistency with
     *              distributed::Vector)
```
either two space or have same indentation